test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -241,33 +200,4 @@ public function invalidVersionStringsProvider() : array
         ];
     }
 
-    /**
```

Comment:
These are to be restored - equality is extremely important in value types, so if we break that invariant we make the system much more complex to reason about